app/domain/sac_imports/huts/huts_row.rb

```diff
@@ -6,23 +6,23 @@
 #  https://github.com/hitobito/hitobito_sac_cas.
 
 module SacImports::Huts
-  class HutsRow
-    def self.can_process?(row)
-      row[:verteilercode] == 4000 && group_type(row).present? &&
+  class HutsRow < Row
+    def can_process?
+      row[:verteilercode] == 4000 && group_type.present? &&
         row[:contact_navision_id] != "00001000"
```

Comment:
In den neueren Exports haben wir keine Navision IDs mit führenden Nullen bekommen. Vermutlich wird dieser Vergleich deshalb keinen Einfluss haben?